LiskHQ/lisk-sdk
# Add monitoring and reportMisbehavior plugin setups - Closes #5876 and #5291

### What was the problem?

This PR resolves #5876 and resolves #5291 

### How was it solved?

- Create packages for `monitor-plugin` and `report-misbehavior-plugin`
- Add them to test/test_app
- Add default configs that matches the requirement
- For report-misbehavior plugin, DB setup is also added

### How was it tested?

- Run test/test-app
- Build should pass
